packages/riipen-ui/src/components/RadioButton.jsx

```diff
@@ -0,0 +1,135 @@
+import clsx from "clsx";
+import PropTypes from "prop-types";
+import React from "react";
+
+import ThemeContext from "../styles/ThemeContext";
+import withClasses from "../utils/withClasses";
+
+import Typography from "./Typography";
+
+const RadioButton = ({
+  checked,
+  classes,
+  disabled,
+  label,
+  prefix: Prefix,
+  prefixProps,
+  suffix: Suffix,
+  suffixProps,
+  ...other
+}) => {
+  const theme = React.useContext(ThemeContext);
+
+  return (
+    <div
+      className={clsx(
+        classes,
+        "root",
+        checked && "checked",
+        disabled && "disabled"
+      )}
+    >
+      <label
```

Comment:
+ does the label need to wrap everything?

Reply:
I use the label instead of wrapping everything in a div (I could change it but then it would be exactly the same except all the styles applied to the div instead of the label with the label also inside the div. I think this is the cleanest way to do it